packages/fx-core/src/plugins/resource/aad/constants.ts

```diff
@@ -145,6 +146,16 @@ export class Messages {
     telemetry: Messages.getEventName("post-provision"),
   };
 
+  static readonly StartGenerateArmTemplates: Messages = {
```

Comment:
We can keep these messages now, but I'm wondering whether each resource plugin needs to show scaffolding arm template messages. cc @MuyangAmigo for awareness